full-service/src/json_rpc/account_key.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020-2021 MobileCoin Inc.
+
+//! API definition for the Account Key object.
+
+use serde_derive::{Deserialize, Serialize};
+
+/// The AccountKey contains a View keypair and a Spend keypair, used to
+/// construct and receive transactions.
+#[derive(Deserialize, Serialize, Default, Debug, Clone)]
+pub struct AccountKey {
+    /// String representing the object's type. Objects of the same type share
+    /// the same value.
+    pub object: String,
```

Comment:
It definitely feels weird using a String field as a type hint, but I guess that's the state of type systems in JSON :(